client/js/table-model.js

```diff
@@ -16,6 +16,21 @@ class TableModel {
   setValue(location, value) {
     this.data[this._getCellId(location)] = value;
   }
+
+  getSum(column) {
+    let sum = [];
```

Comment:
This is not a good variable name.  This variable is going to be collecting numbers (that's why it's an array). It's not the sum -- it's the numbers that will be added to make the sum.